source/_docs/z-wave/installation.markdown

```diff
@@ -223,6 +223,14 @@ $ ls /dev/cu.usbmodem*
 
 If your device path changes when you restart, see [this guide](http://hintshop.ludvig.co.nz/show/persistent-names-usb-serial-devices/) on fixing it.
 
+### {% linkable_title Device not starting up correctly %}
```

Comment:
It would be more sensible to replicate the wording from the Hass.io specific section here

Reply:
Addressed